src/lib/constant_mappings.cpp

```diff
@@ -139,4 +154,15 @@ const boost::bimap<DataType, std::string> data_type_to_string =
                  return map;
                });
 
+const std::unordered_map<DataType, JitDataType> data_type_to_jit_data_type = {{DataType::Int, JitDataType::Int},
```

Comment:
If the only difference to the regular data types is that you added `bool`, maybe we should simply add `bool` to `DataType`?

Or does it have some special meaning other than a data type?

Reply:
It is nothing special. I tried adding a data type `Bool` with mixed results. If I add it as data type `uint8_t`, there are no problems. Adding it as `bool` causes many test to fail: https://ares.epic.hpi.uni-potsdam.de/jenkins/blue/rest/organizations/jenkins/pipelines/Hyrise/pipelines/hyrise/branches/johannes%252Fjit_pr_2_types/runs/7/nodes/28/steps/46/log/?start=0
Apparently the `AllParameterVariant` is not able to handle this properly and returns all `std::strings` as `"1"`.
Does anybody know why this happens and how to fix this?